currency/storage.go

```diff
@@ -167,22 +171,9 @@ func (s *Storage) RunUpdater(overrides BotOverrides, settings *Config, filePath
 	}
 
 	if len(fxSettings) == 0 {
-		log.Warnln(log.Currency, "No foreign exchange providers enabled, setting default provider...")
-		for x := range settings.ForexProviders {
-			if settings.ForexProviders[x].Name != DefaultForexProviderExchangeRatesAPI {
-				continue
-			}
-			settings.ForexProviders[x].Enabled = true
-			settings.ForexProviders[x].PrimaryProvider = true
-			primaryProvider = true
-			log.Warnf(log.Currency, "No valid foreign exchange providers configured. Defaulting to %s.", DefaultForexProviderExchangeRatesAPI)
-			fxSettings = append(fxSettings, base.Settings(settings.ForexProviders[x]))
-		}
-	}
-
-	if len(fxSettings) == 0 {
+		log.Warnln(log.Currency, "No foreign exchange providers enabled, currency conversion will not be available")
 		s.mtx.Unlock()
-		return errNoForeignExchangeProvidersEnabled
+		return nil
```

Comment:
I'm happy with this being nil, I'm only putting this here because I believe it should error elsewhere